frontend/src/entry/overview/EntryCardRolesChipSet.tsx

```diff
@@ -1,22 +1,16 @@
-import Dangerous from '@mui/icons-material/DangerousOutlined'
 import { Chip } from '@mui/material'
+import { useMemo } from 'react'
 import { CollaboratorEntry } from 'types/types'
 
 type EntryCardChipSetProps = {
```

Comment:
My bad for not noticing this the first time around. Should be `EntryCardRolesChipSetProps`

Reply:
Changed